### PR TITLE
Remove Downloads and add Services to top menu

### PR DESCRIPTION
**Pull request by JohanBjerning**

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Deploy preview for *core-website* ready!

Built with commit bec22df5f3785360d2fb0b9556a0ef6383ab706e

https://deploy-preview-495--core-website.netlify.com

File: docs/services/services.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+| [Qlik Associative Engine](./qix-engine/introduction.md) | The powerful associative indexing engine from Qlik and the foundation of Qlik Core. |
+| [Licenses](./licenses.md) | Service required to run Qlik Associative Engine with a paid license. |
+| [Mira](./mira.md) | Qlik Associative Engine discovery service. |
+
+Qlik Core comes with a developer edition that may be used to evaluate,
+develop, or test solutions. Note that it cannot be used for commercial purposes.
+For more information about the license of Qlik Core visit
+[Licensing](/licensing/).
+
+For more information on how to retreive the services vist [Downloads](/downloads/).
```
Comment (carlioth) [inline docs/services/services.md]: visit

File: docs/services/services.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Overview
+
+Qlik Core consists of a core set of supported services, including the Qlik Associative Engine:
```
Comment (carlioth) [inline docs/services/services.md]: I would remove: "including the Qlik Associative Engine:"
Don't think it's needed. Why is QAE included and not the other?

File: docs/services/services.md (hunk 1 above)
Comment (JohanBjerning) [reply docs/services/services.md]: Good catch!

File: docs/services/services.md (hunk 2 above)
Comment (JohanBjerning) [reply docs/services/services.md]: Was copy/paste from Download page, but I agree, removing

**Pull request merged**